ansible/roles/es-mapping/files/mappings/userv3-mapping.json

```diff
@@ -889,6 +889,937 @@
           }
         }
       }
+    },
+    "profileDetails": {
+      "properties": {
+        "photo": {
```

Comment:
Do we need this field to be index, Is this field searchable and whats the significance for searching this field.